docs/concepts/v3/group-chat.md

```diff
@@ -0,0 +1,165 @@
+# Group chat
+
+## Summary
+
+* XMTP V3 is introducing Group Chat based on the MLS [specification](https://www.rfc-editor.org/rfc/rfc9420.html).
+* A group can be anywhere from 1 person to 400.
+* Groups can have administrators with special permissions. Those permissions include:
+  * Add or remove group members.
+  * Set a group title or other metadata.
+  * Pin messages.
+  * Promote members to admins or demote admins.
+  * Update permissions policies.
```

Comment:
From Naomi: @cameronvoell mind validating the permissions part of this? 

Thank you!